.github/workflows/ci-test.yml

```diff
@@ -107,9 +117,6 @@ jobs:
       - uses: actions/checkout@v2
         with:
           ref: ${{ github.event.pull_request.head.sha }}
-      - uses: ./.github/actions/check-machine
```

Comment:
Why are these `check-machine` checks being removed?

Reply:
because the machines have been stable since they were turned on, figured it was time to retire.  we log config elsewhere.

Reply:
I guess my request would be to break this out into a separate commit (and preferably, a separate pull request). 

Including this irrelevant change adds additional 'noise' to the review. I was trying to find a connection between the removal of these scripts and the use of environments for secrets.